src/gui/instrument/InstrumentView.cpp

```diff
@@ -76,4 +78,4 @@ InstrumentTrackWindow * InstrumentView::instrumentTrackWindow()
 
 
 
-} // namespace lmms::gui
\ No newline at end of file
+} // namespace lmms::gui
```

Comment:
Add that newline here, it's a warning on GitHub.

Reply:
I already did add a new-line.

Reply:
Oops mb
I looked at the before and typed that message.